### ISSUE TITLE
Bug: export failure when trying dump undefined `page.html`

### ISSUE THREAD

**Issue by sandersiim** [auth0/auth0-deploy-cli#654]
### Description

Running `export` command fails if there is a `page` with undefined `html`.

### Reproduction

When using export functionality with `--format directory --export_ids --debug`, the following error can happen when there is a Page without html:

```
2022-09-16T08:26:10.203Z - debug: TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received undefined
    at Object.writeFileSync (node:fs:2200:5)
    at /Users/sander.siim/auth0-fetcher/node_modules/auth0-deploy-cli/lib/context/directory/handlers/pages.js:69:36
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/Users/sander.siim/auth0-fetcher/node_modules/auth0-deploy-cli/lib/context/directory/handlers/pages.js:63:15)
    at Generator.next (<anonymous>)
    at /Users/sander.siim/auth0-fetcher/node_modules/auth0-deploy-cli/lib/context/directory/handlers/pages.js:8:71
```

Issue is caused by incorrect Boolean logic here:
```
diff --git a/src/context/directory/handlers/pages.ts b/src/context/directory/handlers/pages.ts
index a346980..4117dcf 100644
--- a/src/context/directory/handlers/pages.ts
+++ b/src/context/directory/handlers/pages.ts
@@ -69,7 +69,7 @@ async function dump(context: DirectoryContext): Promise<void> {
   pages.forEach((page) => {
     var metadata = { ...page };

-    if (page.name !== 'error_page' || page.html !== undefined) {
+    if (page.name !== 'error_page' && page.html !== undefined) {
       // Dump template html to file
       const htmlFile = path.join(pagesFolder, `${page.name}.html`);
       log.info(`Writing ${htmlFile}`);
```

### Environment

> Please provide the following:

- **Version of this library used:** 7.14.3
- **Version of the platform or framework used, if applicable:** any
- **Other relevant versions (language, server software, OS, browser):**
- **Other modules/plugins/libraries that might be involved:**

**Comment by willvedd** [comment]
You're definitely correct about the source of the error. This project has been suffering from template issues for while now, see: #365  #632 #417. The main problem is that the data is inconsistent between templates. I'm going to try and take a stab at applying more descriptive types anywhere we use `fs.writeFileSync()` to avoid writing undefined to disk.

**Comment by willvedd** [comment]
Thanks for opening this ticket. This has been addressed with [7.15.1](https://github.com/auth0/auth0-deploy-cli/releases/tag/v7.15.1). I'm hoping it's the last time the `TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string...` error pops up.
